packages/@uppy/aws-s3-multipart/src/index.test.js

```diff
@@ -52,28 +52,32 @@ describe('AwsS3Multipart', () => {
         }),
         completeMultipartUpload: jest.fn(() => Promise.resolve({ location: 'test' })),
         abortMultipartUpload: jest.fn(),
-        prepareUploadParts: jest.fn(() => {
+        prepareUploadParts: jest.fn(() => new Promise((resolve) => {
```

Comment:
Yes, although I generally only use `async` if I also `await`. This is a bit more explicit in that we need to return a Promise, which is quite unrevealing with `async () =>`.

Reply:
IMO `async () =>` is very clear because an async function will always return a promise, no matter what. I think that `new Promise((resolve) => { resolve() })` should only be used when really needed, e.g wrapping callback or event emitter functions etc.